src/Metalama.Patterns.Immutability/ImmutableAttribute.cs

```diff
@@ -1,10 +1,10 @@
 ﻿// Copyright (c) SharpCrafters s.r.o. See the LICENSE.md file in the root directory of this repository root for details.
 
+using Metalama.Extensions.Validation;
```

Comment:
This should be implemented without the Validation namespace, by using `query...Where(...).ReportDiagnostic`